tests/integration_test.rs

```diff
@@ -80,8 +80,93 @@ fn identity_map() {
              excusable);
 }
 
+#[test]
+#[cfg(not(target_os = "macos"))]
+fn with_functions() {
+    // Ignored on macOS since atos doesn't support -f
+
+    let target = env::current_exe().unwrap();
+    let mut debug = target.clone();
+
+    // On macOS, symbols are kept in a separate .dSYM file
+    if cfg!(target_os = "macos") {
```

Comment:
True. My hope is that eventually we'll find a way to run that test under macOS too, and I kept it in there assume that we'd need to perform the same hacks when that was the case. But I guess you're right that we should rather just take them out for now.